pydy/system.py

```diff
@@ -462,18 +463,37 @@ def generate_ode_function(self, generator='lambdify', **kwargs):
 
         """
 
-        if 'specified' in kwargs:
-            kwargs.pop('specified')
-            print("User supplied 'specified' kwarg was disregarded.")
+        try:
```

Comment:
Yes, please make this backwards compatible. There is a lot of code in the wild that would like to get a warning about this minor change.